### PR TITLE
Add WCF 4.7.2 Dockerfiles

### PR DESCRIPTION
**Pull request by yujayee**

### CODE REVIEW

File: 4.7.2-windowsservercore-ltsc2016/runtime/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM microsoft/aspnet:4.7.2-windowsservercore-Itsc2016
```
Comment (mcy94w) [inline 4.7.2-windowsservercore-ltsc2016/runtime/Dockerfile]: ltsc2016(all lower case)

Comment (yujayee) [reply 4.7.2-windowsservercore-ltsc2016/runtime/Dockerfile]: @mcy94w Thanks, just fixed it.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 # Supported Windows Server 2016 amd64 tags
 
-* 4.7.1, latest ([wcf-docker/4.7.1/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7.1/Dockerfile))
-* 4.7 ([wcf-docker/4.7/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7/Dockerfile))
-* 4.6.2 ([wcf-docker/4.6.2/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.6.2/Dockerfile))
+* 4.7.2-windowsservercore-ltsc2016 ([wcf-docker/4.7.2-windowsservercore-ltsc2016/runtime/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7.2-windowsservercore-ltsc2016/runtime/Dockerfile))
+* 4.7.1 ([wcf-docker/4.7.1-windowsservercore-ltsc2016/runtime/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7.1-windowsservercore-ltsc2016/runtime/Dockerfile))
+* 4.7 ([wcf-docker/4.7-windowsservercore-ltsc2016/runtime/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7-windowsservercore-ltsc2016/runtime/Dockerfile))
+* 4.6.2 ([wcf-docker/4.6.2-windowsservercore-ltsc2016/runtime/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.6.2-windowsservercore-ltsc2016/runtime/Dockerfile))
```
[the remark this answers is not in the document]
Comment (yujayee) [reply README.md]: @zhenlan Please see my latest commit. I've updated "-windowsservercore-ltsc2016" tags. Images with tag "4.6", "4.7", "4.7.1", "4.7.2" and "latest" are multi-arch images, so they support both windows versions.

**Comment by yujayee** [comment]
@Lxiamail @zhenlan I don't have permission to merge this pr. Could any of you help me merge it, or give me write access? Thanks!

**Pull request merged**